### PR TITLE
Remove Profile Icon from Checkout Page

### PR DESCRIPTION
**Pull request by Ap0c**
# Plan

We want to remove the profile icon from the checkout page, while leaving it intact for the rest of the site. This includes removing the identity menu from this page.

Involves building on #1368.

# Changes

- Remove control navigation template from the simple header, we're now not using anything from it (`simpleHeader.scala.html`).
- No need for passing `mainSiteLink` through to control navigation now that we don't use it (`controlNavigation.scala.html`).
- Shifted identity menu code into a submodule (`main.js`, `identityPopup.js`, `identityPopupDetails.js`, `setup.es6`, `identityPopup.spec.js`).
- Added presence of simple header to guardian page object, and check for it before trying to initialise identity menu code (`simpleHeader.scala.html`, `setup.es6`).

# Screenshots

**Before**:

with-profile

**After**:

without-profile

**Normal Header Outside Checkout**:

normal-page

@svillafe @rupertbates @rtyley @JustinPinner

### CODE REVIEW

File: frontend/app/views/fragments/global/controlNavigation.scala.html [hunk 1]
```diff
@@ -75,16 +74,14 @@
...
-                <span class="control__item">
-                    @fragments.inlineIcon("g-mark", List("icon-inline--large"))
-                </span>
-                <span class="control__item">
-                    <span class="control__info">the guardian</span>
-                </span>
-            </a>
-        </li>
-    }
+    <li class="nav-control__item nav-control__item--last hidden-mobile">
```
Comment (rupertbates) [inline frontend/app/views/fragments/global/controlNavigation.scala.html]: Shouldn't this code be deleted now?

Comment (Ap0c) [inline frontend/app/views/fragments/global/controlNavigation.scala.html]: Ah, no, sorry, I should have made that clearer; this code isn't specific to the simple header. It was always shown in the main header, I just [added a flag](https://github.com/guardian/membership-frontend/pull/1368/files#diff-a50ebb05028e3a9ba0785d1ba9dbc24b) so we could have the option of hiding it for the simple one.

However, now that the simple header no longer uses anything from that file, the flag is no longer necessary 😄.

**Comment by Ap0c** [comment]
Looks to be working ok...

**Supporter Page Without Profile Icon:**

supporter-flow

**Successful Partner Sign-Up (also works for Supporter):**
partner-success

Will investigate why post-prod tests are failing (again).

**Comment by Ap0c** [comment]
Temporarily broke post-deploy tests by deleting the display name element. Fixed in #1399.

**Pull request merged**